### ISSUE TITLE
Training code questions

### ISSUE THREAD

**Issue by xxheyu** [CodeGoat24/Pref-GRPO#1]
Thanks for your excellent work. And I have some questions about the training code.

1. In line 787 of fastvideo/train_flux_pref_grpo.py, "get_tokenizer" is not defined
2. Why reward model is none?

**Comment by CodeGoat24** [comment]
In `finetune_prefgrpo_flux.sh`, [UnifiedReward-Think](https://huggingface.co/CodeGoat24/UnifiedReward-Think-qwen-7b) is used as Pairwise Preference Reward Model.

In `finetune_unifiedreward_flux.sh`, [UnifiedReward](https://huggingface.co/collections/CodeGoat24/unifiedreward-20-models-68b7c99ab70ff81184c70270) is used as Pointwise Scoring Reward Model.
